dev-python/qdarkstyle/qdarkstyle-2.7.ebuild

```diff
@@ -0,0 +1,22 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{5,6,7,8} )
+
+inherit distutils-r1
+
+MY_PN=QDarkStyle
+
+DESCRIPTION="A dark style sheet for QtWidgets application"
+HOMEPAGE="https://github.com/ColinDuquesnoy/QDarkStyleSheet"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${MY_PN}-${PV}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="dev-python/helpdev[${PYTHON_USEDEP}]"
+
```

Comment:
Looking at upstream, I see they have some test cases you, then potential for `doc` and `examples` content. Could you look into implementing these in this ebuild?

Reply:
```
develop? ( dev-python/qtsass[${PYTHON_USEDEP}]
			dev-python/watchdog[${PYTHON_USEDEP}] )
examples? ( dev-python/PyQt5[${PYTHON_USEDEP}]
			dev-python/pyside2[${PYTHON_USEDEP}]
			>=dev-python/QtPy-1.7 )
```

I seem to be missing some dependencies to add these. I found pyside2 [here](https://gpo.zugaina.org/dev-python/pyside2). I found no qtsass ebuilds but I suppose I could write one, but repoman is complaining `develop` is a invalid USE flag and I'm unsure what to do about that. I did add `doc`.

Reply:
You can introduce new local USE flag by editing `metadata.xml` file, although you'd first need to understand what upstream means with this `develop` flag and perhaps adjust that in the ebuild correctly.

Reply:
> although you'd first need to understand what upstream means with this develop flag

I can only take a wild guess from the description of qtsass: "Compile SASS files to Qt stylesheets". I think that the develop flag somehow enables something that allows you to create stylesheets like qdarkstyle from 'SASS files' (whatever that may be).

Reply:
> I seem to be missing some dependencies to add these. I found pyside2 here.

TIL, PySide2 and PyQt5 are basically the same thing and QDarkStyle has example files for both (and more). Therefore I have just added `examples? ( dev-python/QtPy )`, leaving PySide out because it is not essential, and it still possible to run the same examples with PyQt5.